src/app/main/cache/cache.service.ts

```diff
@@ -12,21 +12,21 @@ interface CacheContent {
     providedIn: 'root'
 })
 /**
- * Cache Service is an observables based in-memory cache implementation
+ * cache Service is an observables based in-memory cache implementation
  * Keeps track of in-flight observables and sets a default expiry for cached values
  * @export
  * @class CacheService
  */
 export class CacheService {
+    readonly defaultMaxAge: number = 3600000;  // 3600000ms => 1 Stunde
```

Comment:
should we keep readonly class members as all caps with underscore? I'm quite used to that formatting.

Reply:
I think we decided to use underscore in private variables only. 
The ESLint rule for variable names is always **camelCase**. 
**PascalCase** is not allowed or we have to add a specific rule...

Reply:
@mdelez Do you mean in this case it should be like

```typescript
readonly _DefaultMaxAge: number
```

OR

```typescript
readonly _defaultMaxAge: number
```

Reply:
This could be interesting:
<https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/naming-convention.md>

Reply:
I'm talking about UPPER_CASE for `readonly` members. The migration tool changed `DEFAULT_MAX_AGE` to `defaultMaxAge`. I agree on keeping the leading underscore for private variables. Apparently this a source of contention within the JS community however.

Reply:
🙈 Yes, that is what I have now also seen and understood 🤪

Reply:
I added some naming convention options to the eslint configuration (932ddcf). Now it should support our naming conventions from ui-lib design document.